caluma/caluma_workflow/serializers.py

```diff
@@ -226,34 +226,11 @@ def create(self, validated_data):
 
     class Meta:
         model = models.Case
-        fields = ["workflow", "meta", "parent_work_item", "form", "context"]
-
-
-class SaveCaseSerializer(CaseSerializer):
-    @transaction.atomic
-    def create(self, validated_data):
-        self.send_event(
-            events.pre_create_case, case=None, validated_data=validated_data
-        )
-        instance = super().create(validated_data)
-        self.send_event(events.post_create_case, case=instance)
-        return instance
-
-    @transaction.atomic
-    def update(self, instance, validated_data):
-        self.send_event(
-            events.pre_create_case, case=instance, validated_data=validated_data
-        )
-        instance = super().update(instance, validated_data)
-        self.send_event(events.post_create_case, case=instance)
-        return instance
```

Comment:
FYI this has been completely removed since it's IMO wrong to fire pre and post **create** events in an update mutation.